compiler/rustc_query_system/src/ich/hcx.rs

```diff
@@ -26,20 +27,13 @@ fn compute_ignored_attr_names() -> FxHashSet<Symbol> {
 pub struct StableHashingContext<'a> {
     definitions: &'a Definitions,
     cstore: &'a dyn CrateStore,
+    sess: &'a Session,
```

Comment:
This is only added for `assert_default_hashing_controls`, right? I'd prefer to add a more targeted field here that just contains the information we need to do the assertion. Making the session available might encourage accessing more an more information in the future.